optimum/intel/openvino/configuration.py

```diff
@@ -895,7 +907,7 @@ def to_nncf_dict(self) -> Dict[str, Any]:
             "gptq": self.gptq,
             "lora_correction": self.lora_correction,
             "backup_mode": backup_mode,
-            **self.kwargs,
+            **kwargs,
```

Comment:
I think you can just update `result` dictionary with `advanced_parameters` if self.statistics_path is defined. So no need to copy `kwargs`